src/lib/bluetooth/bluetoothbackend.cpp

```diff
@@ -36,6 +36,14 @@ BluetoothBackend::BluetoothBackend(QObject *parent)
 {
```

Comment:
If I am not mistaken we disable squash and merge commits on our repositories. This was decided on one of the compliance office meetings and documented by @DunderRoffe . Could you verify this please?